ui/app/AppLayouts/Profile/controls/AccountDelegate.qml

```diff
@@ -0,0 +1,22 @@
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+
+
+StatusListItem {
+    property var account
+    title: account.name
+    subTitle: account.address
+    image.source: "..."
+    image.isIdenticon: true
+    width: parent.width
+    leftPadding: 0
```

Comment:
can remove this as it is 0 anyway?

Reply:
this is not 0 by default, this is why I set it

Reply:
my bad then